### PR TITLE
mes-9919-useTemplate

### PR DESCRIPTION
**Pull request by RLCorp**
## Description
activated useTemplate
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

**Review by andrewsetterfield — APPROVED** [review]

File: mock/angular-mocks/nav-params.mock.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { NavParams } from '@ionic/angular';
 
 @Injectable()
 export class NavParamsMock extends NavParams {
-    get<T = any>(param: string): T {
-        return super.get(param);
-    }
+  get<T = any>(param: string): T {
```
Comment (sianhedley) [inline mock/angular-mocks/nav-params.mock.ts]: this shouldn't be needed to be cased as any as T is generic

File: mock/angular-mocks/router-mock.ts [hunk 2]
```diff
@@ -3,15 +3,11 @@ export class RouterMock {
 
   routerState = { root: '' };
 
-  navigate = jasmine.createSpy('navigate')
-    .and
-    .returnValue(Promise.resolve(true));
+  navigate = jasmine.createSpy('navigate').and.returnValue(Promise.resolve(true));
```
Comment (sianhedley) [inline mock/angular-mocks/router-mock.ts]: for these you can simply use resolveTo takes out the need for the returnValue(Promise.resolve(true)) - same for all instances where promise.resolve is used

File: mock/ionic-mocks/storage.mock.ts [hunk 3]
```diff
@@ -3,29 +3,15 @@ import { Storage } from '@ionic/storage-angular';
 
 @Injectable()
 export class StorageMock {
+  clear = jasmine.createSpy('clear').and.returnValue(Promise.resolve());
```
Comment (sianhedley) [inline mock/ionic-mocks/storage.mock.ts]: sme comment as before on changing the promise.resolve to resolveTo

File: ngrx-devtool-proxy/remote-devtools-connection-proxy.ts [hunk 4]
```diff
@@ -7,7 +7,7 @@ export class RemoteDevToolsConnectionProxy implements ReduxDevtoolsExtensionConn
 
   subscribe(listener: (change: any) => void): any {
     const listenerWrapper = (change: any) => {
-      console.log(`change: `, change);
+      console.log('change: ', change);
```
Comment (sianhedley) [inline ngrx-devtool-proxy/remote-devtools-connection-proxy.ts]: should this log be in here

File: mock/angular-mocks/nav-params.mock.ts (hunk 1 above)
Comment (RLCorp) [reply mock/angular-mocks/nav-params.mock.ts]: addressed in newest commit

File: mock/angular-mocks/router-mock.ts (hunk 2 above)
Comment (RLCorp) [reply mock/angular-mocks/router-mock.ts]: addressed in newest commit

File: ngrx-devtool-proxy/remote-devtools-connection-proxy.ts (hunk 4 above)
Comment (RLCorp) [reply ngrx-devtool-proxy/remote-devtools-connection-proxy.ts]: removed in newest commit

File: mock/ionic-mocks/storage.mock.ts (hunk 3 above)
Comment (RLCorp) [reply mock/ionic-mocks/storage.mock.ts]: Addressed in newest commit

**Review by sianhedley — APPROVED** [review]

**Pull request merged**